### ISSUE TITLE
Make sure all interactive elements are accessible

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#2614]
Make sure there are no `role="presentation”` or `aria-hidden=”true”` attributes on any focusable/interactive elements as those are not accessible to screen reader users

**Comment by CarolineDenis** [comment]
Tested Specify 7 for role="presentation” and aria-hidden=”true”.
No issue founded 
No role="presentation" in the code
All aria-hidden="true" are attributed to spans used for designs
